source/common/common/utility.cc

```diff
@@ -323,6 +323,33 @@ std::vector<absl::string_view> StringUtil::splitToken(absl::string_view source,
   return absl::StrSplit(source, absl::ByAnyChar(delimiters), absl::SkipEmpty());
 }
 
+std::string StringUtil::removeTokens(absl::string_view source, absl::string_view delimiters,
```

Comment:
I know the other functions here aren't commented, but would you mind commenting up this one?

Also I'm with Matt - I'm fine if you want to remove the options you don't need, but whatever options we have should have their own unit tests.

Reply:
They are part of the unit test.  Do you want that test separated so that we have 3 or 5 different ones?